packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,121 @@
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": "",
+    "test:watch": "npm -s test -- --reporter min --watch-extensions ts,tsx --watch"
+  },
+  "dependencies": {
+    "@itwin/imodel-browser-react": "^0.12.1",
+    "@itwin/itwinui-css": "^0.44.1",
+    "@itwin/itwinui-react": "^1.29.2",
+    "@itwin/itwinui-icons-react": "^1.5.1",
+    "classnames": "^2.3.1",
+    "cross-fetch": "^3.1.4",
+    "i18next": "^10.2.2",
+    "immer": "^9.0.7",
+    "react-spring": "^8.0.27",
+    "react-table": "^7.7.0",
+    "redux": "^4.0.3",
+    "simple-react-validator": "^1.6.1"
+  },
+  "devDependencies": {
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/appui-layout-react": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
+    "@itwin/build-tools": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-i18n": "^3.0.0",
+    "@itwin/core-markup": "^3.0.0",
+    "@itwin/core-orbitgt": "^3.0.0",
+    "@itwin/core-quantity": "^3.0.0",
+    "@itwin/core-react": "^3.0.0",
+    "@itwin/eslint-plugin": "^3.0.0",
+    "@itwin/itwinui-icons-react": "^1.5.1",
+    "@itwin/presentation-common": "^3.0.0",
+    "@itwin/presentation-components": "^3.0.0",
+    "@itwin/presentation-frontend": "^3.0.0",
+    "@itwin/webgl-compatibility": "^3.0.0",
+    "@types/i18next": "^8.4.2",
+    "@types/node": "^17.0.13",
+    "@types/react": "^17.0.19",
+    "@types/react-dom": "^17.0.9",
+    "@types/react-redux": "^7.0.1",
+    "@types/react-table": "^7.7.0",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.10.1",
+    "cpx2": "^3.0.0",
+    "eslint": "^7.32.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-react": "^7.25.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "ncp": "^2.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.0",
+    "rimraf": "^3.0.2",
+    "typescript": "~4.3.0"
+  },
+  "peerDependencies": {
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-react": "^3.0.0",
+    "@itwin/presentation-components": "^3.0.0",
+    "@itwin/presentation-frontend": "^3.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.0"
+  },
+  "eslintConfig": {
+    "plugins": [
+      "@itwin"
+    ],
+    "extends": "plugin:@itwin/ui",
+    "rules": {
```

Comment:
please add the following rules, 

      "no-duplicate-imports": "off",
      "@typescript-eslint/consistent-type-imports": "error"